test-aether-integration-module/conf/pip/requirements.txt

```diff
@@ -9,9 +9,7 @@
 #                                                                              		#
 #########################################################################################
 aet.consumer==0.9.1
-aether.client==0.0.0
-aether.mocker==0.0.0
-aether.saladbar==0.0.0
+aether.client==0.9.0
```

Comment:
wondering how is this going to affect new releases...

Reply:
I swapped it for >= in requirement.txt, but there's not way that I can find to get pip_freeze to generate anything but == in its output.

Reply:
You can remove the dependency here, and introduce it in the `requirements_header.txt` file :thinking: